src/url-utils.js

```diff
@@ -76,3 +77,16 @@ export function sanitizeCodeRepositoryUrl(url) {
   }
   return url
 }
+
+export function getIpfsCidFromUri(string) {
```

Comment:
```suggestion
export function getIpfsCidFromUri(uri) {
```

Minor nitpick 😄 .

TIL that `ipfs:<cid>` is actually a technically valid URI.

It also look like [IPFS addressing standard has stayed with `/ipfs/` and are now officially pushing for `ipfs://<cid>`](https://github.com/ipfs/in-web-browsers/blob/master/ADDRESSING.md#notes-on-addressing-with-ipfs), rather than `ipfs:<cid>`.